ansible_collections/azureredhatopenshift/cluster/tasks/create_aro_cluster.yaml

```diff
@@ -4,6 +4,18 @@
   ansible.builtin.include_tasks:
     file: ../../tasks/create_jumphost.yaml
 
+- name: create_aro_cluster | Gather localhost facts
+  # Registering localhost_facts as a workaround for delegation/hostname resolution issues
+  # in `Gathering Facts` task when setting `gather_facts: true` in deployment playbook.
+  ansible.builtin.setup:
+  delegate_to: localhost
```

Comment:
The inventory hostname in the playbook is fake, so I would never expect gather_facts at the playbook level to work... but yeah we need this to determine the arch of the ansible container for computing os_arch.